### PR TITLE
refactor(paystring): Update to use common table and link styles

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

- Use `.basic` table styles
- Make address links green
- Remove lots of code ~160 lines
- Dynamically add external link icon to all links that arent in the footer
- Fix addresses for networks without an explorer acting like links

Related to #165 and #224

### Context of Change

The Paystring page was overlooked when working on #165 and #224.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before
Screen Shot 2022-08-15 at 6 50 12 PM

## After
Screen Shot 2022-08-15 at 6 41 01 PM

### CODE REVIEW

File: src/containers/Footer/footer.scss [hunk 1]
```diff
@@ -46,7 +46,10 @@
     letter-spacing: 0em;
     line-height: 21px;
     text-align: left;
-    text-decoration: none;
+
+    &::after {
```
Comment (shawnxie999) [inline src/containers/Footer/footer.scss:50]: What would be the purpose to this style?

Comment (ckniffen) [reply src/containers/Footer/footer.scss:50]: This prevents the external link icon from displaying.  I will add a comment.

File: src/containers/shared/css/global.scss [hunk 2]
```diff
@@ -152,4 +152,20 @@ a {
   &:hover {
     color: $green-primary;
   }
+
+  &[href^="http://"],
+  &[href^="https://"],
+  &.external {
```
Comment (shawnxie999) [inline src/containers/shared/css/global.scss:158]: is the external class used anywhere? couldnt find it

Comment (ckniffen) [reply src/containers/shared/css/global.scss:158]: I don't think it is.  I can just remove.

**Comment by ckniffen** [comment]
Requires #247

**Review by shawnxie999 — APPROVED** [review]

**Review by JST5000 — APPROVED** [review]
LGTM - Only note is in the example when you click the bottom link it goes to an "Account Missing" page which I'm pretty sure is due to the account having been deleted on the XRPL, but may be worth double checking

**Comment by ckniffen** [comment]
> LGTM - Only note is in the example when you click the bottom link it goes to an "Account Missing" page which I'm pretty sure is due to the account having been deleted on the XRPL, but may be worth double checking

If you access it from the testnet it will do this weirdness (happens in prod).  I think it is tied to this line https://github.com/ripple/explorer/blob/staging/src/containers/PayStrings/PayStringMappingsTable/index.js#L101

It should check if the paystring is MAINNET but the network the app is running against is testnet it needs to link to the main net url.

**Pull request merged**